app/assets/javascripts/discourse/app/components/modal/forgot-password.hbs

```diff
@@ -22,8 +22,9 @@
         autocapitalize="off"
       />
     {{else}}
+      <p> {{i18n "forgot_password.invite"}}</p>
```

Comment:
Is the preceding space deliberate here @chapoi?

Reply:
No :)

Reply:
```suggestion
      <p>{{i18n "forgot_password.invite"}}</p>
```